UKHomeOffice/nmsw-ui
# Test NMSW 1059 and 1057

- Test NMSW-1059 UPT status checks next actions with save and come back later, reupload fal 6 and auth token missing when clicking reupload FAL 6 /save and come back later
- Test NMSW-1057 Filter passenger status with corresponding  response code 
- Test passenger filter with different status combination


## To test
Run with cypress runner
```
npm run cypress:open:dev -- --env MAIL_API_KEY=${MAIL_API_KEY}
```
and choose passenger-status-check.feature
or in cmd line
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/passenger-status-check.feature
```


## Notes
Auth token missing with finish checking passenger fails and will reset after the issue is fixed
[NMSW-1059](https://jira.bics-collaboration.homeoffice.gov.uk/browse/NMSW-1059)
UPT status checks next actions
